### PR TITLE
Update bitcoin.conf

### PR DESCRIPTION
**Pull request by CohibAA**
Add whitebind, whitelist, rpcbind, and 0.10.0 transaction fee changes.

### CODE REVIEW

File: contrib/debian/examples/bitcoin.conf [hunk 1]
```diff
@@ -94,6 +104,14 @@
 #rpcsslcertificatechainfile=server.cert
 #rpcsslprivatekeyfile=server.pem
 
+# Transaction Fee Changes in 0.10.0
```
Comment (luke-jr) [inline contrib/debian/examples/bitcoin.conf]: Rather than group by version it was added in, it makes sense to group by topic IMO.

Comment (laanwj) [inline contrib/debian/examples/bitcoin.conf]: Agree with @luke-jr here. It is good to mention in the comment when a feature was added, but the organization of the file is more useful when the options are grouped by theme.

**Comment by laanwj** [comment]
@CohibAA are you going to update this?

**Comment by CohibAA** [comment]
@luke-jr @laanwj - I agree it is not structured very well.  I don't always follow the btc dev convos closely, so when the update was available it took me a long time to figure out what affects the changes would have and how to best setup my different servers accordingly, that's all.  I would be happy to help clean up documentation on features and changes in general, but I am only a hobbyist dev at best and I'm not sure what logic and structure would be most appropriate for this project.  In other words, I would, but I don't know where to start, and this was my feeble effort to at least make this available.

**Comment by laanwj** [comment]
Just going to merge this, thanks, any kind of (correct) documentation is better than no documentation.

**Pull request merged**